control-plane/pkg/contract/contract.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.26.0
-// 	protoc        v3.17.3
+// 	protoc-gen-go v1.28.1
+// 	protoc        v3.12.4
```

Comment:
I think one of the reason you have that problem is that the versions don't align, you need to have these exact versions for protoc and protoc-gen-go
```
// 	protoc-gen-go v1.26.0
// 	protoc        v3.17.3
```


if I run it on this PR, I get a commit like https://github.com/pierdipi/eventing-kafka-broker/commit/87de9f101a49edfb9cb789ab6e4ddca3349e3431